kzg_prover/README.md

```diff
@@ -1 +1,52 @@
-# TODO
+# Summa V2: Polynomial Encoding Approach
+
+## Motivation
+
+Summa V1 was using a Merkle sum tree (MST) as the main data structure and a cryptographic commitment. MST that has $n$ leaves involves hashing operations over $2n-1$ entries, making it computationally demanding. Additionally, the MST inclusion proofs in Summa V1 have to be wrapped into a ZK-SNARK, making it computationally demanding to generate all of them at once for the entire user base of the Custodian (~100M users).
```

Comment:
Add link to release version of Summa v1.1 when mentioning it

Reply:
I would say "MST that has $n$ leaves involves $2n-1$ hashing operations, making it computationally demanding. Additionally, the MST inclusion proofs in Summa V1 have to be wrapped into a ZK-SNARK, making it **unfeasible** to generate all of them at once for the entire user base of the Custodian (~100M users).